### PR TITLE
Fix time calculation in the PRs script

### PR DESCRIPTION
**Pull request by milancurcic**
While preparing #357 I found a small issue in the PRs script which manifests only in December. This PR fixes it and adds a requirements.txt file to keep track of the script dependencies.

We should also have a short guide on making a newsletter, which is for another PR.

### CODE REVIEW

**Review by zmoon — COMMENTED** [review]
Oops, sorry about that, and thanks for fixing! We could do it with div-rem math and avoid `dateutil`, but that would be less clear.

File: _scripts/prs.py [hunk 1]
```diff
@@ -3,6 +3,7 @@
 Requires PyGithub.
```
Comment (zmoon) [inline _scripts/prs.py:3]: could add `python-dateutil` to this note? or remove the line, as maybe it won't be necessary after the documentation you mentioned is added

File: _scripts/prs.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
...
 """
 import datetime
+from dateutil.relativedelta import relativedelta
```
Comment (zmoon) [inline _scripts/prs.py:6]: convention would be to put this after `import github`, to stress that it is not part of the std lib

Comment (awvwgk) [reply _scripts/prs.py:6]: I recommend to just run a formatter (`black` or similar) over the script.

File: _scripts/prs.py (hunk 1 above)
Comment (milancurcic) [reply _scripts/prs.py:3]: Yes. I've added requirements.txt which tracks dependencies, but we haven't yet documented this. So for now, I think it's good to mention it here, until it's documented elsewhere.

File: _scripts/prs.py (hunk 2 above)
Comment (milancurcic) [reply _scripts/prs.py:6]: Either sounds good. I haven't used formatters before.

Comment (zmoon) [reply _scripts/prs.py:6]: could add a pre-commit config to do it automatically

Comment (milancurcic) [reply _scripts/prs.py:6]: I ran it through black. Let me know if there's anything else.

**Comment by zmoon** [comment]
Maybe the `requirements.txt` should go inside the `_scripts` directory?

**Comment by milancurcic** [comment]
OK, done, let me know if it's good to go.

**Review by zmoon — APPROVED** [review]

**Pull request merged**